pallets/author-inherent/src/lib.rs

```diff
@@ -112,6 +118,7 @@ decl_module! {
 		}
 
 		fn on_finalize() {
+			// Do we still need this now that it is required?
```

Comment:
Not sure, but I believe we still need it to ensure the author is set every block. 

I expect that `DispatchClass::Mandatory` means that inherents that call this must be included in the block (or at least are prioritized relative to non-mandatory). I do not think mandatory means a call to it is mandatory once per block.

Reply:
I agree with that `Mandatory` is not enough to make this check unnecessary. But I think `is_inherent_required` might be enough. See below.